static/etc/system-image/writable-paths

```diff
@@ -52,12 +52,7 @@
 /var/lib/dbus                           auto                    persistent  none        none
 /var/lib/dhcp                           auto                    persistent  none        none
 # cloud-init
-# FIXME: we currently write an empty /writable/system-data/etc/cloud directory
-#        when creating the image.  So setting /etc/cloud to  "persistent"
-#        instead of "synced" will result all writes getting skipped. Once
-#        the image does no longer contains this empty dir this patch can be
-#        reverted.
-/etc/cloud                              auto                    synced      none        none
+/etc/cloud                              auto                    persistent  none        none
```

Comment:
why was `action` set to `none` here? AFAICT, we want `transition` actually, as `action == none` means that we do nothing with the files already in the core20 snap, which means that files critical to cloud-init running such as `ds-identify.cfg` and friends are not installed on writable and thus cloud-init does not work properly when it should

Reply:
Probably a mistake/misunderstanding. Or wishful thinking in my crusade for empty /etc, i.e. that I hoped the defaults would not have to be stored there. Anyway, thanks for fixing this :)